src/gsy_e/models/strategy/forward/pv.py

```diff
@@ -40,11 +52,24 @@ def remove_open_orders(self, market: "ForwardMarketBase", market_slot: DateTime)
         for offer in offers:
             market.delete_offer(offer)
 
-    def post_order(self, market: "ForwardMarketBase", market_slot: DateTime):
-        order_rate = self._order_updaters[market][market_slot].get_energy_rate(
-            self.area.now)
-
-        capacity_percent = self._order_updaters[market][market_slot].capacity_percent / 100.0
+    def remove_order(self, market: "ForwardMarketBase", market_slot: DateTime, order_uuid: str):
+        bids = [bid
+                for bid in market.slot_bid_mapping[market_slot]
+                if bid.buyer == self.owner.name and bid.id == order_uuid]
```

Comment:
This list will always have the length 1 right? Just a minor comment, 
``` python
        for bid in bids:
            market.delete_bid(bid)
```
suggests that there are actually more bids to be deleted. 
The following approach would also work right?
```python
market.delete(bids[0])
```

Reply:
Also, this is the PV, I guess that this is a copy paste issue, but I propose to rename to `offer`.

Reply:
Agreed on both, will fix them, thanks!